config/components/manager/kustomization.yaml

```
@@ -17,4 +17,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: gcr.io/k8s-staging-kueue/kueue
-  newTag: main
+  newTag: v0.4.0
```

```suggestion
  newTag: release-0.4
```

This is the image we publish for every commit in the staging repo